data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2378 @@
+
+		"Magnetoplasma Drive" 2
+		"Hyperdrive"
+
+	gun -96 -3
+		angle -90
+	gun 96 -3
+		angle 90
+	turret -9 -3
+	turret 9 -3
+
+	engine -92 1.5
+	engine 92 1.5
+	engine -88 1.5
+	engine 88 1.5
+	engine -84 1.5
+	engine 84 1.5
+	engine -80 1.5
+	engine 80 1.5
+	engine -76 1.5
+	engine 76 1.5
+	engine -72 1.5
+	engine 72 1.5
+	"reverse engine" -92 -7
+	"reverse engine" 92 -7
+	"reverse engine" -88 -7
+	"reverse engine" 88 -7
+	"reverse engine" -84 -7
+	"reverse engine" 84 -7
+	"reverse engine" -80 -7
+	"reverse engine" 80 -7
+	"reverse engine" -76 -7
+	"reverse engine" 76 -7
+	"reverse engine" -72 -7
+	"reverse engine" 72 -7
+	"steering engine" -92 -7
+		angle 180
+		left
+	"steering engine" 92 -7
+		angle 180
+		right
+	"steering engine" -88 -7
+		angle 180
+		left
+	"steering engine" 88 -7
+		angle 180
+		right
+	"steering engine" -84 -7
+		angle 180
+		left
+	"steering engine" 84 -7
+		angle 180
+		right
+	"steering engine" -80 -7
+		angle 180
+		left
+	"steering engine" 80 -7
+		angle 180
+		right
+	"steering engine" -76 -7
+		angle 180
+		left
+	"steering engine" 76 -7
+		angle 180
+		right
+	"steering engine" -72 -7
+		angle 180
+		left
+	"steering engine" 72 -7
+		angle 180
+		right
+	"steering engine" -92 1.5
+		right
+	"steering engine" 92 1.5
+		left
+	"steering engine" -88 1.5
+		right
+	"steering engine" 88 1.5
+		left
+	"steering engine" -84 1.5
+		right
+	"steering engine" 84 1.5
+		left
+	"steering engine" -80 1.5
+		right
+	"steering engine" 80 1.5
+		left
+	"steering engine" -76 1.5
+		right
+	"steering engine" 76 1.5
+		left
+	"steering engine" -72 1.5
+		right
+	"steering engine" 72 1.5
+		left
+
+	leak "leak" 30 60
+	leak "flame" 30 60
+	explode "tiny explosion" 18
+	explode "small explosion" 18
+	explode "medium explosion" 15
+	"final explode" "final explosion large"
+
+	description "The Melodikos is a large passenger transport built on the same propulsion and structural frame as the Koryfi and Avlaki, but with a vastly enlarged solar array, as well as a spacious centrifuge habitat. Once used as an interplanetary cycler, it is now used for mass transit between the stars."
+	description "While not as modular as the Koryfi and Avlaki designs, the Melodikos shares the same solar array and structural frame as the Harmonikos. Refits to swap the centrifuge for a set of cargo pods are uncommon due to the work involved, but still possible."
+	description "While its standard set of magnetoplasmadynamic thrusters give it an acceleration generously described as sluggish, the Melodikos' constant thrust flight profile allows it to reach an impressive cruising velocity given enough time."
+
+
+# Freighters
+
+ship "Avlaki"
+	sprite "ship/avgi avlaki"
+		"frame rate" 30
+	thumbnail "thumbnail/avgi avlaki"
+	attributes
+		category "Light Freighter"
+		"cost" 1860000
+		"shields" 600
+		"hull" 1800
+		"required crew" 2
+		"bunks" 6
+		"mass" 180
+		"drag" 3
+		"heat dissipation" 0.72
+		"fuel capacity" 900
+		"cargo space" 240
```

Comment:
An Avlaki can fit enough outfit expansions so that it has have the same cargo space as a Koryfi... and 6 more passenger bunks, on practically the same loadout. This probably isn't worth worrying about, but I do find it really really funny- Just waste 600k credits and you too can make a people-freighter! (The Harmonikos does not have the same quirk).